Neos.Fusion/Tests/Unit/FusionObjects/ArrayImplementationTest.php

```diff
@@ -41,12 +41,12 @@ public function positionalSubElements()
         return [
             [
                 'Position end should put element to end',
-                ['second' => ['__meta' => ['position' => 'end']], 'first' => []],
+                ['second' => ['__meta' => ['position' => 'end']], 'first' => ['__meta' => []]],
```

Comment:
These fixtures are a bit weird now, but IMO the whole unit test is a bit weird already because it exposes quite a lot of internal logic.. this could be easily covered by a functional test with real fusion fixture.